.scripts/phpcs.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# we need this script to fail if any command fails
+# when a command fails the variable `err` will turn 1
+err=0
+trap 'err=1' ERR
```

Comment:
You don't need use trap for control exit code (in case of the error) because if a bash command fails travis receive that and exit with fail.

Reply:
you can use `set -e` to exit on errors this stops on the first command which fails then